ui/src/api/api.ts

```diff
@@ -12,6 +13,12 @@ import {
   PaperIdWithEntityCounts,
 } from "./types";
 
+const token = cookie.parse(document.cookie)["readerAdminToken"];
```

Comment:
How does one set this adminToken locally? Would they open the cookie editor in the browser dev tools? I wonder if it we possible to let us set the reader admin token through environment variables?

Reply:
> How does one set this adminToken locally? Would they open the cookie editor in the browser dev tools?

Right setting it in the browser dev tools is the easiest place to start. Can just add a new cookie by hand with key `readerAdminToken` and value the secret value we can share over slack or what have you.

> I wonder if it we possible to let us set the reader admin token through environment variables?

Do you mean setting it in the server or the browser? If the latter, browsers don't have access to system environment variables.